Terraform/alb.tf

```diff
@@ -1,8 +1,8 @@
 # ALB
 resource "aws_alb" "dev_pro_alb" {
   name            = var.alb_name
-  subnets         = [aws_subnet.dev_pro_public_subnets[0].id, aws_subnet.dev_pro_public_subnets[1].id]
-  security_groups = [aws_security_group.dev_pro_sg_public.id]
+  subnets         = [module.vpc.public_subnet_ids[0], module.vpc.public_subnet_ids[1]]
```

Comment:
`subnets = module.vpc.public_subnet_ids` - ?

Reply:
In this block I specify public_subnet_ids from outputs of vpc-module.

Reply:
module.vpc.public_subnet_ids[0], module.vpc.public_subnet_ids[1] are in the same list which is module.vpc.public_subnet_ids why you added them separately?